dot/simswap/util/reverse2original.py

```diff
@@ -1,30 +1,49 @@
 #!/usr/bin/env python3
 
 import cv2
+import kornia as K
 import numpy as np
 import torch
 import torch.nn as nn
+from kornia.geometry import transform as ko_transform
 from torch.nn import functional as F
 
 
-def encode_segmentation_rgb(segmentation, no_neck=True):
+def isin(ar1, ar2):
```

Comment:
this? https://pytorch.org/docs/stable/generated/torch.isin.html#torch-isin

Reply:
@Ghassen-Chaabouni what pytorch version are you using?

Reply:
`1.9.0+cu111` the one that we use for dot

Reply:
i see. best to upgrade the dependency then.